src/pspm_simple_qa.m

```diff
@@ -73,7 +73,33 @@
 filt = range_filter & slope_filter;
 d(filt) = data(filt);
 
+% write epochs to mat if missing_epochs_filename option is present
+if isfield(options, 'missing_epochs_filename')
```

Comment:
Can you add something like ` && ischar(options.missing_epochs_filename)` ? And if it's not can you send a warning please ? Usually we do not trust the user :sweat_smile: